### PR TITLE
Fix/resize for mobile

### PR DESCRIPTION
**Pull request by asthas**
rescaled everything to play on mobile
The grid offset still needs to be fixed.

### CODE REVIEW

File: CodeBenders/Assets/Scripts/ProjectileGenerator.cs [hunk 1]
```diff
@@ -79,13 +79,15 @@ public class ProjectileGenerator : MonoBehaviour
         float gravityScale)
     {
         // include a GUID in the name so no name collision could be possible
-        var projectile = new GameObject($"Projectile ({Guid.NewGuid()})", 
+        var projectile = new GameObject($"Projectile ({Guid.NewGuid()})",
             new []{typeof(CircleCollider2D), typeof(SpriteRenderer), typeof(Rigidbody2D)});
         // tag the projectile
         projectile.tag = projectileTag;
+        //scale the projectile
+        projectile.transform.localScale = new Vector3(2, 2, 2);
```
Comment (late-in-autumn) [inline CodeBenders/Assets/Scripts/ProjectileGenerator.cs:87]: Works well for a hotfix. But after midterm I would still suggest making the scale factor a proper parameter (see 19e362e27681469bfb4834b32b800e88a9188553 for that).

**Review by late-in-autumn — DISMISSED** [review]
Works. But for mobile devices, there are still a few build settings that we have to change. If you desire, I can change them now and push to this branch.

**Comment by late-in-autumn** [comment]
Realized that now grid boundry is broken and one can't drag the blocks properly. Requesting @bharatht96 for help on this.

**Comment by bharatht96** [comment]
> Realized that now grid boundry is broken and one can't drag the blocks properly. Requesting @bharatht96 for help on this.

Just added commit for this. Please review.

Also see comment in whatsapp group regarding target object sizes.

**Review by niya11 — DISMISSED** [review]
Looks Good

**Comment by late-in-autumn** [comment]
Closing this PR in favorite of #36 which contains other bugfixes.

**Pull request closed without merging**